recipe/patches/0003-customize-setpy.patch

```diff
@@ -0,0 +1,46 @@
+From 3be2fa4f6b4b85fcd94ba01dd6ac0b5b29f432b8 Mon Sep 17 00:00:00 2001
+From: ngam <67342040+ngam@users.noreply.github.com>
+Date: Fri, 4 Feb 2022 16:16:54 -0500
+Subject: [PATCH] customize setpy
```

Comment:
For example, this whole patch should be dropped. Rather the exact same effect should be achieved by a properly-rebased `0004-loosen-requirements.patch`

Reply:
Yes, but my goal was to set up a totally new patch here so that we can tune all requirements on the fly. Properly-rebasing 0004 could've achieved that, yes, but that would have necessarily meant I had a full idea of what I wanted to achieve, which of course, I didn't :/